personal_trainer/demo/account_analytic_line.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="account_analytic_line_1" model="account.analytic.line">
+        <field name="employee_id" ref="hr.employee_admin" />
+        <field name="timesheet_invoice_type">billable_fixed</field>
+        <field name="name">/</field>
+        <field name="date" eval="(datetime.today() - timedelta(days=1)).strftime('%Y-%m-%d')"/>
+        <field name="project_id" ref="project_project_3"/>
+        <field name="task_id" search="[('sale_line_id', '=', ref('sale_order_line_1'))]"/>
+        <field name="product_uom_id" ref="uom.product_uom_hour" />
+        <field name="unit_amount">2.0</field>
+        <field name="department_id" ref="hr.dep_administration" />
+    </record>
+    <record id="account_analytic_line_2" model="account.analytic.line">
+        <field name="employee_id" ref="hr.employee_admin" />
+        <field name="timesheet_invoice_type">billable_fixed</field>
+        <field name="name">/</field>
+        <field name="date" eval="(datetime.today() - timedelta(days=1)).strftime('%Y-%m-%d')"/>
+        <field name="project_id" ref="project_project_3"/>
+        <field name="task_id" search="[('sale_line_id', '=', ref('sale_order_line_1'))]"/>
```

Comment:
This doesn't work. Maybe remove the project_id?

Reply:
It works I have checked this

Reply:
No, because it is counted in the template project, which doesn't make sense
![image](https://github.com/user-attachments/assets/c2620f6c-6df1-43df-9509-a1f9c06a5551)

Reply:
![image](https://github.com/user-attachments/assets/17b637fe-25b8-4f9f-b9de-2a65d652a584)